HW3.py

```diff
@@ -0,0 +1,49 @@
+from pyspark.ml import Pipeline
+from pyspark.sql import SparkSession
+
+from split_column_transform import SplitColumnTransform
+
+appName = "PySpark Example - MariaDB Example"
+master = "local"
+spark = SparkSession.builder.appName(appName).master(master).getOrCreate()
+
+sql = """
+SELECT DATE(g.local_date) AS game_date, s.game_id,
+        s.batter, AVG(s.avg) OVER (ORDER BY DATE(g.local_date)
```

Comment:
you'll lose points on this calculation. Apparently we dont want to take the average of all the averages for that game.